mmv1/third_party/terraform/resources/resource_compute_instance.go

```diff
@@ -641,6 +641,29 @@ func resourceComputeInstance() *schema.Resource {
 					},
 				},
 			},
+			"advanced_machine_features": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Computed:    true,
```

Comment:
Does this need to be marked as Computed?

Reply:
@upodroid Yeah just confirming if you've found a case where the API returns this block when it isn't specified.

Reply:
If none of the child values are computed, I imagine this parent map also doesn't need to be computed. 

I would be really surprised if sending this empty parent field to the API was treated differently than not sending it.

Reply:
Doesn't need to be computed. Fixed